resources/icons/20px/heart.svg

```diff
@@ -1,14 +1,10 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- Generator: Adobe Illustrator 18.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
-<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-	 viewBox="0 0 20 20" enable-background="new 0 0 20 20" xml:space="preserve">
-<g id="heart_4_">
-	<g>
-		<path fill-rule="evenodd" clip-rule="evenodd" d="M20,6.25C20,3.3505,17.6495,1,14.75,1C12.647,1,10.8381,2.2394,10,4.025
-			C9.1619,2.2394,7.353,1,5.25,1C2.3505,1,0,3.3505,0,6.25C0,6.3344,0.0087,6.4166,0.0126,6.5H0.0045
-			c0.0018,0.0596,0.0139,0.1229,0.0173,0.183c0.0051,0.0614,0.0103,0.1225,0.0174,0.1832c0.087,1.0278,0.4012,2.1379,1.0297,3.3071
-			c1.5225,2.839,4.2216,4.9777,8.9307,8.8116c4.709-3.8339,7.4082-5.9741,8.9307-8.8116c0.6261-1.1646,0.9401-2.2706,1.0286-3.2952
-			c0.0084-0.0698,0.0144-0.14,0.02-0.2105C19.982,6.6125,19.9931,6.5545,19.9948,6.5h-0.0074C19.9913,6.4166,20,6.3344,20,6.25z"/>
-	</g>
-</g>
-</svg>
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@pkwi this is broken
![image](https://user-images.githubusercontent.com/464822/38265342-73ee0728-372a-11e8-9dd2-0726002af70c.png)

Reply:
maybe replace the 20px one with the 16px version scaled to 20px?